src/mh-core.jl

```diff
@@ -41,8 +41,8 @@ used if `chain_type=Chains`.
 types are `chain_type=Chains` if `MCMCChains` is imported, or 
 `chain_type=StructArray` if `StructArrays` is imported.
 """
-struct MetropolisHastings{D} <: Metropolis
-    proposal::D
+mutable struct MetropolisHastings{D} <: MHSampler
+    proposal :: D
```

Comment:
AFAIK this should be
```suggestion
    proposal::D
```
according to the Turing style guide?